### ISSUE TITLE
Brilliant implementation

### ISSUE THREAD

**Issue by csoutham** [svnwa/laravel-strapi#2]
Much prefer this implementation over the official one, but does the reset cache route work?
I can see it in the config, but can't see what route it would need to hit.
Can you help please?

**Comment by diakoptis** [comment]
I agree. The implementation is so Laravelish. I love it. Also I cannot find the route :D 

@csoutham maybe you need to implement it by yourself.
